tools/eslint-plugin-cdk/test/rules/invalid-cfn-imports.test.ts

```diff
@@ -0,0 +1,74 @@
+import { ESLint } from 'eslint';
```

Comment:
I actually disagree with @nija-at's earlier comment here. I think this should be folded back in with the `fixtures.test.ts`. IMO, the behavior should be that if an `.expected` file exists, it's checked as it is today, if an `.error` file exists, it does what you're doing here, and if both exist, and error is thrown.

Reply:
Good point, I think @nija-at's main point was that we should actually check the content of the error message. I've re-combined the tests.